structfields.go

```diff
@@ -84,26 +86,33 @@ func (sf *structFields) parse(t reflect.Type, mapper FieldMapperFunc, prefix str
 			}
 		}
 
-		// The alias name has been used by another field.
```

Comment:
Had to move this lower for the use case of same alias with a different `fieldas`